elide-core/src/main/java/com/yahoo/elide/core/EntityBinding.java

```diff
@@ -116,10 +117,12 @@
     public final ConcurrentHashMap<String, Class<?>> fieldsToTypes = new ConcurrentHashMap<>();
     public final ConcurrentHashMap<String, String> aliasesToFields = new ConcurrentHashMap<>();
     public final ConcurrentHashMap<Method, Boolean> requestScopeableMethods = new ConcurrentHashMap<>();
+    public final ConcurrentHashMap<AccessibleObject, Map<String, Object>> attributeArgumets = new ConcurrentHashMap<>();
```

Comment:
The entity dictionary should not store argument values directly (as those are tied to a request).  The dictionary would only know the argument name and type.

Reply:
Rather than store a Map<String, Object> - we might want to store a Set<Argument> and create a new Argument type for readability.